CHANGES

```diff
@@ -54,6 +54,7 @@ Bugs fixed
   set to "description"
 * #8419: html search: Do not load ``language_data.js`` in non-search pages
 * #8454: graphviz: The layout option for graph and digraph directives don't work
+* #8131: linkcheck: Too Many Redirects on HEAD request is treated as broken
```

Comment:
Up to you, I think the message could explain the issue (the web server misbehaves and always redirects to the same location on HEAD), and what the solution is: try a GET request.

Suggested wording:

```suggestion
* #8131: linkcheck: Use GET when HEAD requests cause Too Many Redirects, to accommodate infinite redirect loops on HEAD
```